api/concept/create.py

```
@@ -1,34 +1,39 @@
+    await asyncio.gather(
+        database.init(request, response),
+        post.init(request, response)
+    )
+    # DECLARATION -> ARGUMENT CHECKS    
+    if not post.checks: raise SUG.ERR[0]
+    # DECLARATION -> ARGUMENT RELATIONSHIP
+    if not (post.exists("Ken", "Kes", "Kde") == [True, True, True]): raise SUG.ERR[1]
+    # DECLARATION -> USER AUTHENTIFIED WITH PERMISSIONS
+    if not (database.validate and database.user["Urole"] >= SUG.PER["concept"]["create"]): raise SUG.ERR[2]
+    # DECLARATION -> QUERY
+    return JSONResponse(content = await database.request(
```

Ruff (0.11.9)

`JSONResponse` may be undefined, or defined from star imports

(F405)